src/store/index.ts

```diff
@@ -1,9 +1,11 @@
 import { configureStore } from '@reduxjs/toolkit';
 import { appReducer } from './app/reducer';
+import { agenciesReducer } from '../components/AgenciesPage/reducers/agenciesReducer';
 
 export const store = configureStore({
   reducer: {
     app: appReducer,
+    agencies: agenciesReducer,
```

Comment:
there can be used combineReducers

Reply:
Yes
Agree
I think to add it after Volodimir's or yours PR
But I'we added